src/components/masthead/masthead.js

```diff
@@ -175,9 +175,9 @@ class Masthead extends React.Component {
       riUrl = 'https://access.redhat.com/documentation/en-us/red_hat_managed_integration/1/html-single/release_notes/';
     } else {
       gsUrl =
-        'https://access.redhat.com/documentation/en-us/red_hat_managed_integration/2.0/html-single/getting_started_with_red_hat_managed_integration_2.0/';
+        'https://access.redhat.com/documentation/en-us/red_hat_managed_integration/2/html-single/getting_started_with_red_hat_managed_integration_2/';
```

Comment:
Can this be set as a configuration property / environment setting in a future release so we don't have to update this every time it changes?

Reply:
Yep, that's the plan going forward... for the 2.0 release, the URL was inconsistent with 1.0 so it didn't make sense, but as long as they stick with this version they're using for 2.1 we should be able to automate it.